armsrc/lfops.c

```diff
@@ -1813,6 +1849,39 @@ void EM4xWriteWord(uint32_t flag, uint32_t Data, uint32_t Pwd) {
 	LED_A_OFF();
 	cmd_send(CMD_ACK,0,0,0,0,0);
 }
+
+void EM4xProtect(uint32_t flag, uint32_t Data, uint32_t Pwd) {
+	
+	bool PwdMode = (flag & 0xF);
```

Comment:
Why `& 0xF` ?

Reply:
Lol,  I guess I didn't look at the original code close enough...  because I just copied and tweaked the writeword function and it was like that there...  

You are correct that it can be reduced to 1 not 0xf...

Reply:
fixed